packages/@pollyjs/core/src/polly.js

```diff
@@ -3,7 +3,7 @@ import { MODES, assert } from '@pollyjs/utils';
 import { version } from '../package.json';
 
 import Logger from './-private/logger';
-import Container from './-private/container';
+import Container, { nameOfFactory } from './-private/container';
```

Comment:
iirc rollup has had issues supporting default and named exports. Might need to switch them both to a named export.

Reply:
and our tests still pass with this issue?